pages/common/croc.md

```diff
@@ -0,0 +1,32 @@
+# croc
+
+> Send and recieve files easily and security over any network.
+> More information: <https://github.com/schollz/croc>.
+
+- Send a file or folder:
+
+`croc send {{path/to/file}}`
+
+- Send a file or folder with a specific passphrase:
+
+`croc send --code {{your-own-created-passphrase}} {{path/to/file}}`
+
+- Recieve a file or folder on reciving machine:
+
+`croc {{the-passphrase-to-recieve-a-file}}`
+
+- Send over a custom relay and custom relay:
+
+`croc --relay {{ip-to-relay}} send {{path/to/file}}`
+
+- Recieve over a custom relay and custom relay:
+
+`croc --relay {{ip-to-relay}} {{the-passphrase-to-recieve-a-file}}`
+
+- Host your own croc relay with default ports ("`9009,9010,9011,9012,9013`"):
```

Comment:
```suggestion
- Host your own croc relay with default ports (9009, 9010, 9011, 9012, 9013):
```

Reply:
Hmm, I think the syntax is to have it as a string of comma separated list.
So if one wants to use the command one needs to write `"port,port2,port3,port4"` as a string, and that's the default value in the help menu.
I try to convey this better.

Reply:
I removed the mention of which ports are the default ones.